paymentsheet/src/main/java/com/stripe/android/paymentsheet/wallet/SavedPaymentMethodsSheetResult.kt

```diff
@@ -0,0 +1,48 @@
+package com.stripe.android.paymentsheet.wallet
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.ExperimentalSavedPaymentMethodsApi
+import com.stripe.android.paymentsheet.model.PaymentOption
+
+@ExperimentalSavedPaymentMethodsApi
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+sealed class SavedPaymentMethodsSheetResult {
+    /**
+     * The customer selected a payment method
+     * @param selection, the [PaymentOptionSelection] the customer selected from the
+     * [SavedPaymentMethodsSheet]
+     */
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Up to you on these, but I think we'll need to add a bunch of experimental annotations to these at some point.